src/System.Private.CoreLib/src/System/Reflection/Emit/ILGenerator.cs

```diff
@@ -159,6 +136,7 @@ internal virtual void RecordTokenFixup()
             m_RelocFixupList[m_RelocFixupCount++] = m_length;
         }
 
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
```

Comment:
This use of AggressiveInlining is too much code bloat. This method is called 25x and each of those places will have copy of both this method and UpdateStackSize since UpdateStackSize is marked with AggressiveInlining as well.

Reply:
https://github.com/dotnet/coreclr/pull/23833/commits/c7b3b52d8e26e68d70333099744353afeb20da74

See comment in review above.
Are the changes satisfactory? Can't really think of other ways to further reduce bloat.

Reply:
This method should not have AggressiveInlining. Yes, it will make the microbechmarks a bit slower, but that's fine.

Reply:
Removed in https://github.com/dotnet/coreclr/pull/23833/commits/b3c7e1a5f448afa738abf671ffb9d9a7a5af5ad7